static/app/components/modals/demoEndModal.tsx

```diff
@@ -0,0 +1,195 @@
+
+function DemoEndingModal({tour, closeModal, orgSlug}: Props) {
+  const api = new Client();
+
+  const routeContext = useRouteContext();
+  const {router} = routeContext;
+
+  function togglePanel(panel: SidebarPanelKey) {
+    SidebarPanelStore.togglePanel(panel);
+  }
+
+  let cardTitle, body, guides, location;
+  switch (tour) {
+    case 'issues':
+      cardTitle = 'Issues Tour';
+      body = 'completing the Issues tour';
+      guides = ['issues_v3', 'issue_stream_v3'];
+      location = `/organizations/${orgSlug}/issues/`;
+      break;
+    case 'performance':
+      cardTitle = 'Performance Tour';
+      body = 'completing the Performance tour';
+      guides = ['performance', 'transaction_summary', 'transaction_details_v2'];
+      location = `/organizations/${orgSlug}/performance/`;
+      break;
+    case 'releases':
+      cardTitle = 'Releases Tour';
+      body = 'completing the Releases tour';
+      guides = ['releases_v2', 'react-native-release', 'release-details_v2'];
+      location = `/organizations/${orgSlug}/releases/`;
+      break;
+    case 'tabs':
+      cardTitle = 'Check out the different tabs';
+      body = 'checking out the different tabs';
+      guides = ['sidebar_v2'];
+      location = `/organizations/${orgSlug}/projects/`;
+      break;
+    default:
+  }
+
+  const sandboxData = window.SandboxData;
+  const url = sandboxData?.cta?.url || 'https://sentry.io/signup/';
+
+  async function handleRestart() {
+    await Promise.all(
+      guides.map(guide =>
+        api.requestPromise('/assistant/', {
+          method: 'PUT',
+          data: {guide, status: 'restart'},
+        })
+      )
+    ).then(() => {
+      if (tour === 'issues') {
+        localStorage.setItem('issueGuide', '1');
+      }
+      if (tour === 'tabs') {
+        localStorage.setItem('sidebarGuide', '1');
+      }
+
+      if (closeModal) {
+        closeModal();
+      }
+
+      fetchGuides();
+
+      const redirectUrl = new URL(location, window.location.origin);
+      redirectUrl.searchParams.append('referrer', 'demo_task');
+      navigateTo(redirectUrl.toString(), router);
+    });
+  }
+
+  const handleMoreTours = () => {
+    if (closeModal) {
+      closeModal();
+    }
+    togglePanel(SidebarPanelKey.OnboardingWizard);
+  };
+
+  return (
+    <EndModal>
+      <CloseButton
+        size="zero"
+        onClick={() => {
+          if (closeModal) {
+            closeModal();
+          }
+        }}
+        aria-label={t('Close')}
+        icon={<IconClose size="xs" />}
+      />
+      <Body>
+        <h2> {t('Tour Complete')} </h2>
+      </Body>
+      <ModalTask title={tct('[title]', {title: cardTitle})} />
+      <Body>
+        {tct(
+          'Thank you for [body]. Learn about other Sentry features by starting another tour.',
+          {body}
+        )}
+      </Body>
+      <ButtonContainer>
+        <a href={url} target="_blank" rel="noopener noreferrer">
```

Comment:
Use `ExternalLink`

Reply:
Actaully since it's wrapping a button, it should use `Button` with the [`external`](https://github.com/getsentry/sentry/blob/a66d78d30e14f5e86645f1d6a91b296ffb7a09b6/static/app/components/button.tsx#L61-L64) prop.